apps/frontend/components/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Header = () => {
+  return <h2>Securing Safe Food</h2>;
```

Comment:
Is there a reason for using `h2` instead of `h1` if this is going to be the site heading?

Reply:
I see, I didn't think much of it at the time and i'll make the change right now.